Backpack-SwiftUI/FlowStackView/Classes/BPKFlowStackView.swift

```diff
@@ -75,22 +75,33 @@ public struct BPKFlowStackView<Data: Collection, Content: View>: View where Data
         }
     }
     
-    func computeRows() -> [[Data.Element]] {
-        var rows: [[Data.Element]] = [[]]
+    private struct Item: Hashable, Identifiable {
+        let id: Int
```

Comment:
If we change it to this, we don't need any index calculation on creation of the item, and it will still be unique
```suggestion
         let id: UUID = UUID()
```

Reply:
We do need it, it's not about having an actual ID, but something to keep track of the index of items